readme.txt

```diff
@@ -3,7 +3,7 @@
 Contributors: automattic, aaronfc, burtrw, donnapep, fjorgemota, gabrielcaires, gikaragia, imranh920, jakeom, luchad0res, merkushin, m1r0, onubrooks, renathoc, yscik
 Tags: lms, eLearning, teach, online courses, woocommerce
 Requires at least: 6.0
```

Comment:
Since we have an L-2 sliding window support policy for WordPress, we should also bump the required version to 6.1.

Reply:
And we could bump the minimum in `php.yml` too. 🙂

Reply:
Updated in https://github.com/Automattic/sensei/pull/7057/commits/ef9550e54f23fd85fd6b9b09a8acdc462507078d. 👍